pkg/grapicmd/context.go

```diff
@@ -45,51 +36,67 @@ type Config struct {
 	}
 }
 
+type BuildConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type BuildConfig should have comment or be unexported

Reply:
fixed in 3041c36